### PR TITLE
feat: added getAssetId and getTargetHash

### PR DESCRIPTION
**Pull request by isaackps**
# In this PR:
- [x] added getAssetId function
- [x] added getTargetHash function

### CODE REVIEW

File: src/shared/utils/utils.ts [hunk 1]
```diff
@@ -92,6 +92,31 @@ export const getMerkleRoot = (document: any): string => {
...
+    case isWrappedV3Document(document):
+      return document.proof.targetHash;
+    default:
+      throw new Error(
+        "Unsupported document type: Only can retrieve target hash from wrapped OpenAttestation v2 & v3 documents."
+      );
+  }
+};
+
+export const getAssetId = (document: any): string => {
```
Comment (rjchow) [inline src/shared/utils/utils.ts:108]: if you extract the code for checking if it is a transferable asset you can simplify this function down to 

if (isTransferableAsset(document) { return getTargetHash(document) }
else { throw error }

this way you don't need to maintain two identical code in getTargetHash and here for retrieving the targetHash

Comment (rjchow) [reply src/shared/utils/utils.ts:108]: other than that lgtm

**Review by rjchow — COMMENTED** [review]
lgtm other than that bit on isTransferableAsset

File: src/shared/utils/utils.ts (hunk 1 above)
Comment (isaackps) [reply src/shared/utils/utils.ts:115]: ok refactored accordingly.

**Review by rjchow — APPROVED** [review]

**Pull request merged**